templates/hooks/version/sdk_create_pre_build_request.go.tpl

```diff
@@ -0,0 +1,18 @@
+res := &svcsdk.ListVersionsByFunctionInput{}
+res.FunctionName = desired.ko.Spec.FunctionName
+var list *svcsdk.ListVersionsByFunctionOutput
+list, err = rm.sdkapi.ListVersionsByFunctionWithContext(ctx, res)
+if err != nil {
+	return nil, err
+}
+bigList := list.Versions
```

Comment:
Maybe `bigList` should be called something more specific, like `versionsList` or `previousVersions` (Or something that makes more sense, for when it's used in the `sdk_create_pre_set_output.go.tpl` file)

Reply:
Makes sense, I went with `versionList`. We're getting list of Versions for `LayerVersion` resource and assigned it to variable named `versionList` only, made sense to use the same naming for this case too. Let me know if we should keep the naming different.